libraries/Microsoft.Bot.Schema/SharePoint/QuickViewData.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Schema.SharePoint
+{
+    /// <summary>
+    /// SharePoint Quick View Data object.
+    /// </summary>
+    public class QuickViewData
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="QuickViewData"/> class.
+        /// </summary>
+        public QuickViewData()
```

Comment:
This should actually be an empty interface, because the data property can be an object with an arbitrary amount of properties, so long as the properties correspond to something in the Adaptive card body